.deepsource.toml

```diff
@@ -0,0 +1,12 @@
+version = 1
```

Comment:
What does adding this file do? Will it run something (DeepSource?) automatically through some GitHub actions?

Reply:
[DeepSource](https://deepsource.io/) works as an additional service that you don’t have to run yourself. All you have to do is just integrate DeepSource in the repo and create a configuration for DeepSource (`.deepsource.toml` file) and leave the rest on DeepSource :sunglasses: 

Here are the steps to do it:
- Install DeepSource on your repository [here](https://deepsource.io/signup).
- Activate analysis [here](https://deepsource.io/gh/CN-UPB/DeepCoMP/).